core/src/test/java/co/infinum/goldfinger/ValidateUtilsTest.java

```diff
@@ -29,7 +30,7 @@ public void auth_invalid_emptyTitle() {
             .negativeButtonText(NEGATIVE_BUTTON_TEXT)
             .description(DESCRIPTION)
             .subtitle(SUBTITLE)
-            .deviceCredentialsAllowed(true)
+            .allowedAuthenticators(BiometricManager.Authenticators.BIOMETRIC_WEAK & BiometricManager.Authenticators.DEVICE_CREDENTIAL)
```

Comment:
I might be wrong here, but it would make more sense to me if logical or is used instead of logical and.
Not sure how it works though. If those constants are some numbers they will cancel each other out unless their bits are both 1 on a specific location. I can see that official docs also use `or` [here](https://developer.android.com/training/sign-in/biometric-auth#declare-supported-authentication-types)

Reply:
Yes you are right, this was a result of experimenting and not deleting the test code. Good catch!